archinstall/lib/global_menu.py

```diff
@@ -197,6 +197,14 @@ def _get_menu_options(self) -> list[MenuItem]:
 			),
 		]
 
+		if arch_config_handler.args.skip_boot:
```

Comment:
Do we really want to omit the entire boot menu entry? I was under the impression the `--skip-boot` will enable the "No bootloader" option to be selected?

Reply:
Oh, my tired brain read:

> "Should this maybe just be a "No bootloader" option"

As: 

"maybe there shouldn't be a boot loader option"

I think the `Bootloader.NO_BOOTLOADER` option to be selected makes sense, I'll put the menu entry back :)

Reply:
All good, this should make the change much simpler :)

Reply:
Should be back now, waiting for ISO build to complete so I can test a full install.